app/controllers/application_controller.rb

```diff
@@ -222,6 +226,15 @@ def add_provider_count_cookie
     logger.error "Error setting the provider_count cookie: #{e.class.name}, #{e.message}"
   end
 
+  def add_interrupt_pages
+    return unless FeatureService.enabled?("rollover.can_edit_current_and_next_cycles") || FeatureService.enabled?("rollover.show_next_cycle_allocation_recruitment_page")
+
+    InterruptPageAcknowledgement.where(user_id: current_user["user_id"], recruitment_cycle_year: Settings.current_cycle.next).all.each do |acknowledgement|
```

Comment:
This looks good. Hard to get my head around sending the next cycle year but I'm not sure I've any better idea.

Reply:
I was thinking an alternative would be naming the pages per year, eg `rollover_2022`. But then it would mean we would need to keep updating the enum on TTAPI, or just drop validation altogether